### PR TITLE
Dominos Pizza Docs

### PR DESCRIPTION
**Pull request by wardcraigj**
**Description:**
Adding documentation for dominos pizza platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#10379

## Checklist:

  - [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/dominos.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Now you can use the Dominos service to order pizza within your automations:
+
+```yaml
+- service: dominos.order
+  data:
+    order_entity_id: dominos.medium_pan
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/dominos.markdown]: Please use the configuration tasks for this part of the documentation.
For more information see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/dominos.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- **country_code** (*Required*): 'ca' or 'us', depending on your location
+- **first_name** (*Required*): Your first name
+- **last_name** (*Required*): Your last name
+- **email** (*Required*): Your email address
+- **phone** (*Required*): Your phone number
+- **address** (*Required*): Your delivery address
+- **show_menu** (*Optional*): Dumps product codes from your nearest stor into your log (for use in with the custom panel)
```
Comment (frenck) [inline source/_components/dominos.markdown]: Typo: `stor` -> `store`

File: source/_components/dominos.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+title: "Dominos Pizza"
+description: "Instructions how to setup Dominos Pizza ordering within Home Assistant."
+date: 2017-11-05 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: dominos.png
+ha_category: Other
+ha_version: 0.57
```
Comment (frenck) [inline source/_components/dominos.markdown]: `0.57` is already out, just like `0.58`. You might want to update this to `0.59`.

File: source/_components/dominos.markdown [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Dominos Pizza"
+description: "Instructions how to setup Dominos Pizza ordering within Home Assistant."
```
Comment (frenck) [inline source/_components/dominos.markdown]: Grammar: `Instructions how to` -> `Instructions on how to`

**Comment by wardcraigj** [comment]
Thanks @frenck! Updated!

File: source/_components/dominos.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/dominos.markdown]: @wardcraigj You need to remove this header as well

**Comment by frenck** [comment]
@wardcraigj The header of the configuration variables need to be removed as well.

Furthermore, the Travis build fails. Please check it out and make sure it passes.

**Comment by frenck** [comment]
Thank you @wardcraigj for updating! 
Can be merged as soon as the parent PR gets merged.

**Pull request merged**